code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -626,15 +631,18 @@ pub mod pallet {
 					owner,
 					asset_id: pool_asset,
 					reward_configs: initial_reward_config,
+					start_block,
 					end_block,
 					lock,
 					share_asset_id,
 					financial_nft_asset_id,
 				} => {
+					// now <= start_block < end_block
```

Comment:
```suggestion
					// now < start_block < end_block
```

Reply:
as per vim's comment below

Reply:
(should also rename the start block error)

Reply:
c65b1e665cf3712c7cdede65c1cd5b9a926c46e3